allensdk/brain_observatory/gaze_mapping/__main__.py

```diff
@@ -29,23 +30,68 @@
 )
 
 
-def repackage_input_args(parser_args: dict) -> dict:
-    """Repackage arguments obtained by argschema.
+def load_ellipse_fit_params(input_file: Path) -> Dict[str, pd.DataFrame]:
```

Comment:
Is this change required to convert ares to cm^2

Reply:
No, it's a light refactor.